v0/destinations/google_adwords_remarketing_lists/transform.js

```diff
@@ -133,62 +133,45 @@ const populateIdentifiers = (attributeArray, { Config }) => {
 const createPayload = (message, destination) => {
   const { listData } = message.properties;
 
-  if (!listData.add || !isArray(listData.add)) {
-    throw new CustomError(
-      "[Google_adwords_remarketing_list]::add is not present inside listData. Aborting message.",
-      400
+  let outputPayloads = {};
+  const typeOfOperation = Object.keys(listData);
+  typeOfOperation.forEach(key => {
+    const userIdentifiersList = populateIdentifiers(
+      listData[`${key}`],
+      destination
     );
-  }
-  const userIdentidtifiersList = populateIdentifiers(listData.add, destination);
-  const outputPayload = constructPayload(message, offlineDataJobsMapping);
-  outputPayload.operations = [];
-  // breaking the userIdentiFier array in chunks of 20
-  const userIdentifierChunks = returnArrayOfSubarrays(
-    userIdentidtifiersList,
-    20
-  );
-  // putting each chunk in different create/remove operations
-  userIdentifierChunks.forEach(element => {
-    const operations = {
-      create: {}
-    };
-    operations.create.userIdentifiers = element;
-    outputPayload.operations.push(operations);
+    const outputPayload = constructPayload(message, offlineDataJobsMapping);
+    outputPayload.operations = [];
+    // breaking the userIdentiFier array in chunks of 20
+    const userIdentifierChunks = returnArrayOfSubarrays(
+      userIdentifiersList,
+      20
+    );
+    // putting each chunk in different create/remove operations
+    if (key === "add") {
```

Comment:
Here all the `userIdentifiers` will go to `create` operation and `remove` as we are not mapping which `identifiers` are part of `create` and which are part of `remove`

Example
Input
```
{
    "userId": "user1",
    "anonymousId": "anon-id-new",
    "event": "event1",
    "type": "audienceList",
    "properties": {
        "listData": {
            "add": [
                {
                    "email": "alex@example.com",
                    "phone": "1234567890"
                },
                {
                    "email": "emily@example.com",
                    "phone": "1234567890"
                }
            ],
            "remove": [
                {
                    "email": "alex@example.com",
                    "phone": "1234567890"
                },
                {
                    "email": "emily@example.com",
                    "phone": "1234567890"
                }
            ]
        },
        "enablePartialFailure": true
    },
    "context": {
        "ip": "1.2.3.40",
        "library": {
            "name": "http"
        }
    },
    "timestamp": "2020-02-02T00:23:09.544Z"
}
```

Output
```
{
  create: {
    enablePartialFailure: true,
    operations: [
      {
        create: {
          userIdentifiers: [
            {
              hashedEmail: "6db61e6dcbcf2390e4a46af426f26a133a3bee45021422fc7ae86e9136f14110",
            },
            {
              hashedPhoneNumber: "c775e7b757ede630cd0aa1113bd102661ab38829ca52a6422ab782862f268646",
            },
            {
              hashedEmail: "6599ce4d8f9d3289820e0af8c795c5a06bdb512fe9cd3baaaeb132d35e641b09",
            },
            {
              hashedPhoneNumber: "c775e7b757ede630cd0aa1113bd102661ab38829ca52a6422ab782862f268646",
            },
          ],
        },
      },
    ],
  },
  remove: {
    enablePartialFailure: true,
    operations: [
      {
        remove: {
          userIdentifiers: [
            {
              hashedEmail: "6db61e6dcbcf2390e4a46af426f26a133a3bee45021422fc7ae86e9136f14110",
            },
            {
              hashedPhoneNumber: "c775e7b757ede630cd0aa1113bd102661ab38829ca52a6422ab782862f268646",
            },
            {
              hashedEmail: "6599ce4d8f9d3289820e0af8c795c5a06bdb512fe9cd3baaaeb132d35e641b09",
            },
            {
              hashedPhoneNumber: "c775e7b757ede630cd0aa1113bd102661ab38829ca52a6422ab782862f268646",
            },
          ],
        },
      },
    ],
  },
}
```